### ISSUE TITLE
[com.discord] Discord - Talk, Video Chat & Hang Out with Friends

### ISSUE THREAD

**Issue by bugreportanon** [microg/GmsCore#1374]
**Affected app**
Name: Discord - Talk, Video Chat & Hang Out with Friends
Package id: com.discord

**Describe the bug**
Captcha doesn't appear when logging in, redirecting to the browser allows you to login to discord but the app refuses to move past the verify captcha screen.

**To Reproduce**
1. Login
2. Get stuck on "verify captcha"
3. Redirected to browser and logins properly
4. Return to app to be stuck on "verify captcha"

**Expected behavior**
Able to login to discord app.

**Screenshots**
![Screenshot_20210124-155231_Discord](https://user-images.githubusercontent.com/69066429/105621514-499d9b80-5e5c-11eb-8476-a0b53bb32819.png)

**System**
Android Version: 10
Custom ROM: LineageOS for MicroG 17.1

**microG**
microG Core version: v0.2.17.204714
microG Self-Check results: All ticked

**Additional context**
N/A

**Comment by Vendicated** [comment]
+1

However it seems like you only have to do a captcha during registration or when entering a wrong login. I had the same issue but then noticed I was entering the wrong password, once I entered the proper password I didnt even get this screen

**Comment by bugreportanon** [comment]
I use bitwarden to save my passwords and the passwords are correct yet it's stuck in a loop with the captcha, I was able to sign in through the browser and on PC without issues but not on mobile for some reason.

**Comment by GLaDTheresCake** [comment]
Yeah stuck here in the same way, the "Having issues with CAPTCHA?" button redirects to the website version of discord. It will not redirect back to log into the app for me on either firefox or chromium. So sadly that also seems to not work as a workaround. Open in app in firefox also doesn't auto login the app.

**Comment by GLaDTheresCake** [comment]
Also I would recommend making the name of this issue more generic, the true issue here is the lack of recaptcha support in the GmsCore, making sure you can't do the login flow of apps that depend on it. We shouldn't make an issue for every app that breaks because of this. Recommend something along the lines of "Lack of google captcha support in microG breaking login flow."

**Comment by dantecpu** [comment]
Problem is no more since moving to hcaptcha resolved the issue.

**Comment by Elixir76** [comment]
A workaround is verifying your location by re-logging in on another device on the same network or logging in from a web browser first, then login from the app and if the credentials are correct it'll let you in without asking for captcha

At least this worked for me on my old tablet

**Comment by ale5000-git** [comment]
If I'm not wrong microG added support for captcha some times ago, have you tried if it still doesn't work?
